### ISSUE TITLE
Root strategies/ tree is a stale duplicate that contradicts the deployed stop loss

### ISSUE THREAD

**Issue by KoscheiiB** [IvPalmer/Master-Trader#89]
## Problem

`strategies/` at the repository root holds six Freqtrade strategy files that nothing loads. Every
one is an older, divergent copy of a file in the authoritative
`ft_userdata/user_data/strategies/` tree.

| file | root | authoritative |
|---|---|---|
| `BollingerRSIMeanReversion.py` | 97 | 171 |
| `ClucHAnix.py` | 223 | 225 |
| `ElliotV5.py` | 194 | 196 |
| `MasterTraderV1.py` | 133 | 197 |
| `NASOSv5.py` | 324 | 323 |
| `SupertrendStrategy.py` | 156 | 228 |

None is byte-identical to its counterpart. The tree was last touched by `6364fb8` on 2026-03-12.

The concrete hazard is the stop loss on `MasterTraderV1`:

```
strategies/MasterTraderV1.py:30
    stoploss = -0.15  # 15% — evidence: 5% < 1 daily SD for alts, causes whipsaws on 1h

ft_userdata/user_data/strategies/MasterTraderV1.py:41
    stoploss = -0.05  # Data: 0% of trades recover past -7%, 92% of winners never dip past -3%
```

Both carry an inline evidence comment and they argue opposite conclusions. Someone who greps the
repository for a strategy name finds the root copy first, reads a justified-looking parameter, and
is reasoning about a file the fleet does not load.

## Deletion test

Run against `c3a837b` rather than inferred. This repository runs components as subprocesses and
Freqtrade resolves strategies by name, so an import graph alone would not settle it.

- **Mounts.** All seven strategy bind mounts point at `./user_data/strategies`:
  `ft_userdata/docker-compose.yml:242` and `ft_userdata/docker-compose.prod.yml:64`, `:91`,
  `:116`, `:255`, `:313`, `:414`. None points at the root tree.
- **Imports.** `grep -rn "from strategies\|import strategies" --include='*.py'` returns nothing.
- **Configuration.** No config, shell script, YAML or workflow references the path.
- **Prose.** One reference in the tree, `research/risk-implementation-plan.md:28`, which cites
  `strategies/ClucHAnix.py` for an ADX threshold. That document is itself stale.

## The question, rather than a patch

Two closures and I have no strong view between them, which is why this is an issue and not a
deletion PR:

1. **Delete the tree.** Cleanest, and the history stays in git. The `-0.15` reasoning is
   recoverable from `6364fb8` by anyone who wants it.
2. **Leave it and signpost it.** A short `strategies/README.md` saying the tree is historical and
   naming the authoritative path. Keeps the old parameters visible at the cost of keeping a
   duplicate that will drift further.

If it is deleted, `research/risk-implementation-plan.md:28` should be either updated or explicitly
marked historical in the same change, so the only prose reference does not dangle.

## Why this one is low risk

It is outside every lane you have been working in, it is untouched by any open branch, and both
copies have been cold for six months. Filed under #4 because it is the same theme: what a new
contributor finds first should be what the fleet actually runs.
